### PR TITLE
Update libraries' building docs for native components

### PR DESCRIPTION
**Pull request by gfoidl**
Contributes to https://github.com/dotnet/runtime/issues/70134

For me this information is enough, as now I have a reference to look at on how to update the native components.
It's helpful when iterating on the native components, and to update the testhost with the new native binaries.

In the `src/native/libs` a very short ReadMe.md got added, that links to the more details libraries's build documentation.
I'm not sure if this is striclty needed, but I find it helpful, as when navigating to that folder and I see `build-native.sh` then I'd run this script, and wonder why the testhost isn't updated. With the ReadMe in place, it's a nice remainder on how to build the native side actually.

/cc: @ViktorHofer

### CODE REVIEW

File: src/native/libs/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+For information on how to build, how to update the binplacing of the native artifacts please refer to
+[How to building native components only](../../../docs/workflow/building/libraries/ReadMe.md#how-to-building-native-components-only)
```
Comment (jkotas) [inline src/native/libs/ReadMe.md:2]:
```suggestion
[How to building native components only](../../../docs/workflow/building/libraries/README.md#how-to-building-native-components-only)
```

Comment (jkotas) [reply src/native/libs/ReadMe.md:2]: (the link won't work in the browser when the casing does not match)

Comment (gfoidl) [reply src/native/libs/ReadMe.md]: Thanks!

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Thanks a lot @gfoidl

**Pull request merged**